packages/firestore/lite/src/api/reference.ts

```diff
@@ -49,3 +55,222 @@ export class DocumentReference<T = firestore.DocumentData>
     return new DocumentReference<U>(this.firestore, this._key, converter);
   }
 }
+
+export class Query<T = firestore.DocumentData> implements firestore.Query<T> {
```

Comment:
Stripped down version of https://osscs.corp.google.com/firebase-sdk/firebase-js-sdk/+/master:packages/firestore/src/api/database.ts;l=1432